Bottom: 6bf0c9c3912cdb60e12cc89a479195bd74f8cd1f
Top:    489fcd498cfff2742a6b5f6cf482afa672f079a3
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-16 13:05:25 -0400

Refresh of mpc5200-ac97

---

diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index 13b53c3..77a5f73 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -273,7 +273,7 @@ static struct snd_soc_dai_ops psc_ac97_digital_ops = {
 	.hw_params	= psc_ac97_hw_digital_params,
 };
 
-static struct snd_soc_dai psc_dai_ac97[] = {
+static struct snd_soc_dai mpc5200_dai_ac97[] = {
 {
 	.name	= "mpc5200 AC97 analog",
 	.id	= MPC5200_AC97_ANALOG,
@@ -313,7 +313,7 @@ static struct snd_soc_dai psc_dai_ac97[] = {
 	},
 	.ops 	= &psc_ac97_digital_ops,
 }};
-EXPORT_SYMBOL_GPL(psc_dai_ac97);
+EXPORT_SYMBOL_GPL(mpc5200_dai_ac97);
 
 static int psc_ac97_hw_params(struct snd_pcm_substream *substream,
 				 struct snd_pcm_hw_params *params,
diff --git a/sound/soc/fsl/mpc5200_psc_ac97.h b/sound/soc/fsl/mpc5200_psc_ac97.h
index d028c55..7daba97 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.h
+++ b/sound/soc/fsl/mpc5200_psc_ac97.h
@@ -10,4 +10,6 @@
 #define MPC5200_AC97_ANALOG 0
 #define MPC5200_AC97_DIGITAL 1
 
+extern struct snd_soc_dai mpc5200_dai_ac97[];
+
 #endif /* __SOUND_SOC_FSL_MPC52xx_PSC_AC97_H__ */
